src/openvino.cc

```diff
@@ -227,7 +227,7 @@ ModelState::ParseParameters()
 {
   triton::common::TritonJson::Value params;
   bool status = model_config_.Find("parameters", &params);
-  if (status) {
+  if (status && !params.IsEmpty()) {
```

Comment:
Model configuration will still have `parameters` field with empty content even there are no parameters set in the config.